projects/plugins/wpcomsh/tests/bootstrap.php

```diff
@@ -60,10 +60,40 @@ function _manually_load_plugin() {
 	if ( file_exists( WPMU_PLUGIN_DIR . '/wpcomsh-loader.php' ) ) {
 		return;
 	}
+
 	require_once dirname( __DIR__ ) . '/wpcomsh.php';
 }
 tests_add_filter( 'muplugins_loaded', '_manually_load_plugin' );
 
+/**
+ * Add the various plugins to the active options array.
+ */
+function _manually_load_other_plugins( $active_plugins ) {
+	// Get the JP_MONO_INTEGRATION_PLUGINS env var to include other plugins.
+	$plugins = getenv( 'JP_MONO_INTEGRATION_PLUGINS' );
+	if ( $plugins ) {
+		$plugins = explode( ',', $plugins );
+		require_once ABSPATH . 'wp-admin/includes/plugin.php';
+		$all_plugins = get_plugins();
+		foreach ( $plugins as $plugin ) {
+			// Ignore wpcomsh and jetpack for now.
```

Comment:
wpcomsh I understand, since this _is_ wpcomsh and we probably don't want to activate it twice. But why jetpack?